free-2.md

```diff
@@ -58,11 +82,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+If someone were to work on this, he/she would spend more time in the language aspect of the project, as the language will be unique, and would need a lot of effort to have a basic but different syntax that everyone would understand. I would say an 80% to 20% compared to the systems aspects of the project, as the main focus is designing the language. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+The idea is very big, ultimately a language that every potential and present game developer would use to design games. It is not as ambitious as my first idea because there are similar examples out there that serve the similar needs.
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
```

Comment:
I would love to know more about how we could work to simplify game designing with a DSL, I think this is a totally cool topic. One of my concerns is that by deceasing the complexity of the programing we might also decrease the complexity of the games.